idea/src/META-INF/plugin.xml

```diff
@@ -2180,6 +2180,14 @@
                      language="kotlin"
     />
 
+    <localInspection implementationClass="org.jetbrains.kotlin.idea.inspections.MemberVisibilityCanPrivateInspection"
+                     displayName="Can have 'private' visibility"
```

Comment:
Better to show a more full name here: "Class member can have 'private' visibility"